docs/product-user-guide/src/main/asciidoc/case-management-stages-con.adoc

```diff
@@ -1,10 +1,18 @@
 = Stages
 
-Case management stages are a collection of tasks that are required to be completed in order to reach specific milestones, or to resolve the case. A stage represents a number of activities that are meant to serve certain business purposes. 
+Case management stages are a collection of tasks that are required to be completed in order to reach specific milestones, or to resolve the case. A stage is an ad hoc subprocess that can be defined using the case designer and may include other case management nodes, such as a milestone. A milestone can also be configured as completed when a stage or a number of stages are completed. Therefore, a milestone may be activated or achieved by the completion of a stage, and a stage may include a milestone or a number of milestones.
 
-A stage consists of the following:
+A stage represents a number of activities that are meant to serve certain business purposes. For example, in a patient triage case, the first stage may consist of observing and noting any obvious physical symptoms or a description from the patient of what their symptoms are, followed by a second stage for tests, and a third for diagnosis and treatment. 
```

Comment:
"represents activities" (for concision again)

Reply:
ack.